test/generator/example_jittest.cpp

```diff
@@ -17,59 +17,102 @@ void verify(const Buffer<int32_t> &img, float compiletime_factor, float runtime_
 
 int main(int argc, char **argv) {
     GeneratorContext context(get_jit_target_from_environment());
+    const float runtime_factor = 4.5f;
+
+    // Demonstrate (and test) various ways to use a Stub to invoke a Generator with the JIT.
+
+    {
+        // The simplest way is to just use the Stub's static "generate" method.
+        //
+        // The Generator's Input<>s are specified via a struct that is initialized
+        // via an {initializer-list}, in the order the Input<>s are declared in the Generator.
+        Func f = example::generate(context, {runtime_factor});
+        Buffer<int32_t> img = f.realize(kSize, kSize, 3);
+        verify(img, 1.f, runtime_factor, 3);
+    }
 
     {
-        // Create a Generator and set its Inputs and GeneratorParams.
-        // We could just use initializer-list syntax, but we'll explicitly
-        // set the fields by name for clarity.
+        // Of course, we can fill in the Inputs struct by name if we prefer.
         example::Inputs inputs;
-        inputs.runtime_factor = 1.f;
+        inputs.runtime_factor = runtime_factor;
 
-        // The fields of the GeneratorParams struct are initialized to the
-        // default values specified in the Generator, so we can just omit
-        // any we don't want to change
-        example::GeneratorParams gp;
-        gp.compiletime_factor = 2.5f;
-        gp.enummy = Enum_enummy::foo;
-        // gp.channels = 3;  -- this is the default; no need to set
+        Func f = example::generate(context, inputs);
+        Buffer<int32_t> img = f.realize(kSize, kSize, 3);
+        verify(img, 1.f, runtime_factor, 3);
+    }
 
-        auto gen = example(context, inputs, gp);
+    {
```

Comment:
I don't see this block being all that useful. It demonstrates a rudimental property of C++ as opposed to something particular to this interface. Style wise it might be useful sometimes, but here it is just kinda tricky to read and interrupts the flow of what you are explaining. I don't think it tests anything the above block doesn't.

Reply:
Removed. PTAL, I think we're ready to land this.